Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -512,7 +512,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
                    _("Compare the angle of the specified object."),
                    _("the angle (in degrees)"),
                    _("Angle"),
-                   "res/conditions/direction24.png",
+                   "res/conditions/direction24_black.png",
```

Comment:
should this be rotate too?

Reply:
Same as above: I kept this icon for actions/conditions that only mention the angle